app/models/pipeline_run_stage.rb

```diff
@@ -282,10 +292,12 @@ def db_load_alignment
     version_s3_path = "#{pipeline_run.alignment_output_s3_path}/#{PipelineRun::VERSION_JSON_NAME}"
     pr.version = `aws s3 cp #{version_s3_path} -`
 
-    # only keep species level counts
+    # only keep counts at certain taxonomic levels
     taxon_counts_attributes_filtered = []
+    acceptable_tax_levels = [TaxonCount::TAX_LEVEL_SPECIES]
+    acceptable_tax_levels << TaxonCount::TAX_LEVEL_GENUS if pr.pipeline_branch == 'charles/hit-calling'
     pipeline_output_dict['taxon_counts_attributes'].each do |tcnt|
-      if tcnt['tax_level'].to_i == TaxonCount::TAX_LEVEL_SPECIES
+      if acceptable_tax_levels.include?(tcnt['tax_level'].to_i) && !invalid_genus_call?(tcnt)
```

Comment:
Why are we dropping hits that align poorly at the genus level but very well at a family level?

Reply:
The invalid_genus_call function also catches invalid species calls, doesn't it?

Reply:
Yes, we could choose to display them. I omitted them for now because the current report page design does not include family-level information, only genus and species. So seeing "Non-genus-specific <family name> hits" looked out of place to me.